tests/upgrades/test_virtwho.py

```diff
@@ -18,9 +18,9 @@
 """
 import pytest
 from fauxfactory import gen_string
-from nailgun import entities
 
 from robottelo.cli.host import Host
+from robottelo.cli.simple_content_access import SimpleContentAccess
 from robottelo.cli.subscription import Subscription
 from robottelo.cli.virt_who_config import VirtWhoConfig
```

Comment:
@jyejare  Thank you very much for your sugggestion. Already removed the SimpleContentAccess import. Jitendra, because these usage are for the CLI of Host, Subscription and VirtWhoConfig not API, some of the info get from API and CLI are different, and some API can not support.